libs/ledgerjs/script/crypto-assets-importer/fetch.ts

```diff
@@ -0,0 +1,14 @@
+import axios, { AxiosError } from "axios";
+
+const CRYPTO_ASSETS_BASE_URL = "https://cdn.live.ledger.com/cryptoassets";
+
+export const fetchTokens = async <T>(filename: string): Promise<T | null> => {
+  try {
+    const { data } = await axios.get<T>(`${CRYPTO_ASSETS_BASE_URL}/${filename}`);
```

Comment:
Shouldn't we use the network lib ?

Reply:
`live-network` should be used for code used in the live, here it's a standalone script

Reply:
I checked the network lib, indeed there is app specific logic